README.md

```diff
@@ -198,3 +198,84 @@ func Function(
 	
 }
 ```
+
+### Sync Tool and Operator Types
+
+To switch to using just the runtime client and to remove dependencies on individual operators, the sync tool will periodically copy over types from operator repos to this repo. It works by cloning the operator repo, applying changes in the config file, then copying it into this repo.
```

Comment:
So we clone the whole repo just for one file? Why not just get the single file

Reply:
@achuzhoy We do a sparse checkout but I'll update to mention that